tensorflow_serving/model_servers/model_management_impl.cc

```diff
@@ -0,0 +1,16 @@
+#include "tensorflow_serving/model_servers/model_management_impl.h"
+#include "tensorflow_serving/util/status_util.h"
+
+namespace tensorflow {
+    namespace serving {
+        Status ModelManagementImpl::Load(ServerCore *core,
+                                              const LoadModelConfigurationRequest* request,
+                                              LoadModelConfigurationResponse *response) {
+
+            const Status &status = core->ReloadConfig(request->configuration());
+            const StatusProto &statusProto = ToStatusProto(status);
```

Comment:
nit: please use the style status_proto instead of statusProto for consistency with the rest of our code-base (same in the rest of your PR, if you have any other camel-case variable names)